app/src/main/res/layout/activity_send_request.xml

```diff
@@ -48,14 +48,26 @@
             android:id="@+id/tvRequestEndDate"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="20dp"
             android:layout_marginStart="8dp"
             android:layout_marginTop="16dp"
-            app:layout_constraintEnd_toEndOf="parent"
+            android:layout_marginEnd="8dp"
+            android:hint="@string/end_date_format"
+            app:layout_constraintEnd_toStartOf="@+id/calendar_icon"
             app:layout_constraintHorizontal_bias="0.06"
             app:layout_constraintStart_toEndOf="@+id/textView8"
+            app:layout_constraintTop_toBottomOf="@+id/rgCurrentUserRole" />
+
+        <android.support.v7.widget.AppCompatImageView
+            android:id="@+id/calendar_icon"
```

Comment:
Can you please rename this to be `ivCalendar`? this is short for imageViewCalendar, how we're naming the rest of the elements of the UI.

Reply:
ok